scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -549,6 +552,13 @@ def parse_field(data, token):
             if 'enum' in root and has_default:
                 root['enum'].insert(0, '')
                 root['enum'].sort()
+            if 'type' in root and root['type'] == 'integer':
+                for fld in ['default', 'minimum', 'exclusiveMinimum', 'maximum', 'exclusiveMaximum']:
+                    if fld in root:
+                        try:
+                            root[fld] = int(root[fld])
+                        except ValueError:
+                            raise RuntimeError("found float %s for integer field" % fld)
```

Comment:
I originally put that in as a backstop in case something crazy was happening. When I put it in I was sure hoping that it wouldn't get hit, it didn't, and then I just left it in. The message isn't quite right, that's for sure. I could just take away the try/except, but unfortunately it appears that `int` truncates rather than rounds. Let me take a quick run back through things and see if truncating instead of rounding is a problem.

Reply:
Yeah, int seems to always truncate toward zero.  OK, if the error was attempting to catch a bad string input or something then the `ValueError` catch seems appropriate.  I was just confused since a float input should pass fine.  I'll wait to see what you decide to do with it.  Thanks.